### PR TITLE
added QoL improvements for development

### PR DESCRIPTION
**Pull request by DedsecKnight**
Added following services into `docker-compose.yml`
- Adminer for viewing data stored in SQl database. 
- RedisInsight for viewing data stored in Redis instance.  
- Instructions on how to use the above 2 services.

### CODE REVIEW

File: docs/dbms-guide.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+
+## How to use Adminer
+1. To start with, spin up all containers using the following command: 
+```bash
+$ docker compose --env-file ./.env up --build --force-recreate
+``` 
+
+2. Go to `localhost:8080` and fill out the fields as follow: 
+- Make sure to choose `PostgreSQL for System`
```
Comment (spierceVR) [inline docs/dbms-guide.md]:
```suggestion
- Make sure to choose `PostgreSQL` for System
```

**Review by spierceVR — APPROVED** [review]
I noticed one thing that could be formatted differently for a little more clarity. 

Besides that, I followed the instructions and everything worked for me. Great job Mike!

**Review by Punchwood2003 — APPROVED** [review]
Looks great to me! 

One thing that I don't think was mentioned, and _might_ fall outside the scope of what we are intending to cover in this document is describing how to set up that initial Redis/Postgres DB. This is mainly stemming from the fact that, as of the last time that we discussed this, do not have a single centralized database that we are all communally using, but rather personal DBs that we are using for development. 

Just a thought of something that might be worth including at some point, or maybe something that is better suited for an internalized document. Either way, PR looks great!

**Comment by DedsecKnight** [comment]
> Looks great to me!
> 
> One thing that I don't think was mentioned, and _might_ fall outside the scope of what we are intending to cover in this document is describing how to set up that initial Redis/Postgres DB. This is mainly stemming from the fact that, as of the last time that we discussed this, do not have a single centralized database that we are all communally using, but rather personal DBs that we are using for development.
> 
> Just a thought of something that might be worth including at some point, or maybe something that is better suited for an internalized document. Either way, PR looks great!

The initial Redis/Postgres is already part of the `docker-compose.yaml` file, which will be spun up when running `docker compose up`.

**Pull request merged**